src/module_2/groceries_analysis.md

```diff
@@ -0,0 +1,1811 @@
+```python
+%reset -f
+```
+
+# Importing data
+
+Data has been stored in local using aws cli. 
+
+
+```python
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+import pandas as pd
+import seaborn as sns
+
+from IPython.display import Markdown, display
+
+DATA_PATH = "/home/alejandro/Zrive"
+
+abandoned_carts_df = pd.read_parquet(os.path.join(DATA_PATH, 'abandoned_carts.parquet'))
```

Comment:
TIP: use [patlib](https://docs.python.org/3/library/pathlib.html) module for paths. It is the more modern way to deal with theam than os.path.join